src/atoms/fileActions.test.ts

```diff
@@ -0,0 +1,329 @@
+    });
+
+    expect(leftPane.current.files).not.toContainEqual(fileA);
+  });
+
+  test('should close all opened files with closeAllFilesAtom', () => {
+    const store = createStore();
+    const { result: openFile } = renderHook(() => useSetAtom(openFileAtom, { store }));
+    const { result: closeAllFiles } = renderHook(() => useSetAtom(closeAllFilesAtom, { store }));
+    const { result: leftPane } = renderHook(() => useAtomValue(leftPaneAtom, { store }));
+    const { result: rightPane } = renderHook(() => useAtomValue(rightPaneAtom, { store }));
+
+    const fileA = makeFile('file.txt');
+    const fileB = makeFile('file.pdf');
+
+    act(() => {
+      openFile.current(fileA);
+      openFile.current(fileB);
+    });
+
+    const paneFiles = [...leftPane.current.files, ...rightPane.current.files];
+    expect(paneFiles).toContainEqual(fileA);
+    expect(paneFiles).toContainEqual(fileB);
+
+    act(() => closeAllFiles.current());
+
+    const totalOpenedFiles = leftPane.current.files.length + rightPane.current.files.length;
+    expect(totalOpenedFiles).toBe(0);
+  });
+
+  test('should split (move) file from one tab to the other', () => {
+    const store = createStore();
+    const { result: openFile } = renderHook(() => useSetAtom(openFileAtom, { store }));
+    const { result: leftPane } = renderHook(() => useAtomValue(leftPaneAtom, { store }));
+    const { result: rightPane } = renderHook(() => useAtomValue(rightPaneAtom, { store }));
+    const { result: splitFileToPane } = renderHook(() => useSetAtom(splitFileToPaneAtom, { store }));
+
+    const fileA = makeFile('file.txt');
+
+    act(() => {
+      openFile.current(fileA);
+    });
+
+    expect(leftPane.current.files).toContainEqual(fileA);
+    expect(rightPane.current.files).not.toContainEqual(fileA);
+
+    act(() => {
+      splitFileToPane.current({ fileId: fileA.path, fromPaneId: 'LEFT', toPaneId: 'RIGHT' });
+    });
+
+    expect(leftPane.current.files).not.toContainEqual(fileA);
+    expect(rightPane.current.files).toContainEqual(fileA);
+  });
+
+  test('should make another file active, in same pane, after closing the active', () => {
+    const store = createStore();
+    const { result: openFile } = renderHook(() => useSetAtom(openFileAtom, { store }));
+    const { result: closeFileFromPane } = renderHook(() => useSetAtom(closeFileFromPaneAtom, { store }));
+    const { result: leftPane } = renderHook(() => useAtomValue(leftPaneAtom, { store }));
+
+    const fileA = makeFile('fileA.txt');
+    const fileB = makeFile('fileB.txt');
+    const fileC = makeFile('fileC.txt');
+
+    act(() => {
+      openFile.current(fileA);
+      openFile.current(fileB);
+      openFile.current(fileC);
+    });
+
+    expect(leftPane.current.activeFile).toEqual(fileC);
+
+    act(() => {
+      closeFileFromPane.current({ paneId: leftPane.current.id, fileId: fileC.path });
+    });
+
+    expect(leftPane.current.activeFile).not.toEqual(fileC);
```

Comment:
I think you can remove this line: you just need to check that the activeFile is fileB (which is done on the next line)